content/desktop/synchronized-file-sharing.md

```diff
@@ -105,10 +87,10 @@ Since Docker [acquired Mutagen](https://www.docker.com/blog/mutagen-acquisition/
 
 - You cannot remove a file share instance during the initial synchronization. You have to wait for it to complete before **Delete** has any effect.
 
-- Synchronized file sharing proactively reports temporary issues, which can result in occasional **Conflict** and **Problem** indicators appearing in the GUI during synchronization. These can be ignored. However, if they persist, you can report the issue.
+- Synchronized File Shares proactively reports temporary issues, which can result in occasional **Conflict** and **Problem** indicators appearing in the GUI during synchronization. These can be ignored. However, if they persist, you can report the issue.
 
 - If you switch from WSL2 to Hyper-V on Windows, Docker Desktop needs to be fully restarted.
 
```

Comment:
I would add one more known issue, which is that we don't support POSIX-style Windows paths (which aren't really a standard anyway), so things like `/c/Users/jacob/some/path` won't use synchronized file shares (they'll fall back to normal file sharing). This is tracked by POS-2168, but it won't be fixed before 4.27.  The biggest impact is that users should avoid setting `COMPOSE_CONVERT_WINDOWS_PATHS`, but this isn't on by default and is a legacy setting that most won't use anyway.